src/Features/Core/Portable/QuickInfo/IEmbeddedLanguageQuickInfoProvider.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.EmbeddedLanguages;
+
+namespace Microsoft.CodeAnalysis.QuickInfo
+{
+    internal interface IEmbeddedLanguageQuickInfoProvider : IEmbeddedLanguageFeatureService
+    {
+        /// <summary>
+        /// Gets the <see cref="QuickInfoItem"/> for the position in an embedded language.
+        /// </summary>
+        /// <returns>The <see cref="QuickInfoItem"/> or null if no item is available.</returns>
```

Comment:
might want to mention the token is already known to be a string token containing the appropriate embedded language.